src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -355,39 +355,47 @@
 
 <div class="spinner-container">
     <h2>Spinner</h2>
-    <div>
-        <span>Extra small spinner:</span>
-        <BitSpinner Size="SpinnerSize.XSmall" />
-    </div>
-    <br />
-    <div>
-        <span>Small spinner:</span>
-        <BitSpinner Size="SpinnerSize.Small" />
-    </div>
-    <br />
-    <div>
-        <span>Medium spinner:</span>
-        <BitSpinner Size="SpinnerSize.Medium" />
-    </div>
-    <br />
-    <div>
-        <span>Large spinner:</span>
-        <BitSpinner Size="SpinnerSize.Large" />
+    <div class="spinner-sizes">
+        <h3>Spinner sizes</h3>
+        <div>
+            <div>
+                <label>Extra small spinner</label>
+                <BitSpinner Size="SpinnerSize.XSmall" />
+            </div>
+            <div>
+                <label>Small spinner</label>
+                <BitSpinner Size="SpinnerSize.Small" />
+            </div>
+            <div>
+                <label>Medium spinner</label>
+                <BitSpinner Size="SpinnerSize.Medium" />
+            </div>
+            <div>
+                <label>Large spinner</label>
+                <BitSpinner Size="SpinnerSize.Large" />
+            </div>
+        </div>
     </div>
-    <br />
     <div class="spinner-positioning">
         <h3>Spinner label positioning</h3>
-        <span>Spinner with label positioned below</span>
-        <BitSpinner LabelPosition="SpinnerLabelPosition.Bottom" Label="I am definitely loading..." />
-        <br />
-        <span>Spinner with label positioned above</span>
-        <BitSpinner LabelPosition="SpinnerLabelPosition.Top" Label="Seriously, still loading..." />
-        <br />
-        <span>Spinner with label positioned to right</span>
-        <BitSpinner LabelPosition="SpinnerLabelPosition.Right" Label="Wait, wait..." />
-        <br />
-        <span>Spinner with label positioned to left</span>
-        <BitSpinner LabelPosition="SpinnerLabelPosition.Left" Label="Nope, still loading..." />
+        <div>
+            <div>
+                <label>Spinner with label positioned below</label>
+                <BitSpinner LabelPosition="SpinnerLabelPosition.Top" Label="I am definitely loading..." />
+            </div>
+            <div>
+                <label>Spinner with label positioned above</label>
+                <BitSpinner LabelPosition="SpinnerLabelPosition.Bottom" Label="Seriously, still loading..." />
+            </div>
+            <div>
+                <label>Spinner with label positioned to right</label>
+                <BitSpinner LabelPosition="SpinnerLabelPosition.Left" Label="Wait, wait..." />
+            </div>
+            <div>
+                <label>Spinner with label positioned to left</label>
+                <BitSpinner LabelPosition="SpinnerLabelPosition.Right" Label="Nope, still loading..." />
+            </div>
+        </div>
     </div>
```

Comment:
please do not change the Component.razor file for the components that have their specific demo pages developed.
for example, the BitSpinner component has its own demo page in the components folder.